include/zephyr/display/cfb.h

```diff
@@ -89,6 +89,20 @@ struct cfb_font {
  */
 int cfb_print(const struct device *dev, char *str, uint16_t x, uint16_t y);
 
+/**
+ * @brief Print a string into the framebuffer.
+ * For compare to cfb_print, cfb_draw_text accept non tile-aligned coords
+ * and not line wrapping.
+ *
+ * @param dev Pointer to device structure for driver instance
+ * @param str String to print
+ * @param x Position in X direction of the beginning of the string
+ * @param y Position in Y direction of the beginning of the string
+ *
+ * @return 0 on success, negative value otherwise
+ */
+int cfb_draw_text(const struct device *dev, char *str, int16_t x, int16_t y);
```

Comment:
Would be nice with `const char *str`!?  `cfb_print` should probably also have const parameter IMO.